### PR TITLE
Feanil/ecom worker 16.04

### PR DESCRIPTION
**Pull request by feanil**
@edx/devops @rlucioni @clintonb Please review.

It turns out we didn't make a docker container for ecomworker.  Building it here so we can run the tests on top of it for the 16.04 work.

### CODE REVIEW

**Review by jibsheet — APPROVED** [review]

File: util/parsefiles_config.yml [hunk 1]
```diff
@@ -27,3 +27,4 @@ weights:
   - elasticsearch: 7
   - docker-tools: 3
   - tools_jenkins: 8
+  - ecomworker: 4
```
Comment (rlucioni) [inline util/parsefiles_config.yml:30]: What is this and why is it 4?

File: docker/build/ecomworker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# To build this Dockerfile:
+#
+# From the root of configuration:
+#
+# docker build -f docker/build/discovery/Dockerfile .
```
Comment (rlucioni) [inline docker/build/ecomworker/Dockerfile]: I 👀  your copypasta. Replace discovery with ecomworker.

**Review by clintonb — APPROVED** [review]

File: docker/build/ecomworker/Dockerfile (hunk 2 above)
Comment (clintonb) [inline docker/build/ecomworker/Dockerfile]: s/discovery/ecomworker

**Review by rlucioni — APPROVED** [review]

File: util/parsefiles_config.yml (hunk 1 above)
Comment (mikedikan) [reply util/parsefiles_config.yml:30]: i'm guessing an ordering mechanism for build ordering, but i'm curious too

Comment (feanil) [reply util/parsefiles_config.yml:30]: This is the amount of time it takes to build the docker container during testing.  It's used to evenly shard out all the test builds across the 4 travis shards we use.

**Comment by feanil** [comment]
Thanks everyone, copy pasta consumed.  Will merge once the tests pass again.

File: util/parsefiles_config.yml (hunk 1 above)
Comment (rlucioni) [reply util/parsefiles_config.yml:30]: Got it, thanks!

**Review by mikedikan — APPROVED** [review]

**Pull request merged**